core/src/main/java/io/aiven/kafka/tieredstorage/fetch/cache/DiskBasedChunkCacheMetrics.java

```diff
@@ -28,7 +28,7 @@
 import org.apache.kafka.common.utils.Time;
 
 class DiskBasedChunkCacheMetrics {
-    private static final String METRIC_GROUP = "chunk-cache-disk";
+    private static final String METRIC_GROUP = "chunk-cache-disk-metrics";
```

Comment:
Should we make the rename first and then rename the metrics as well into `disk-chunk-cache-metrics` and etc.?

Reply:
Good idea, let's focus on #455  and applied suggestion as fixup commit already